shared/devices/device-revoke/index.js

```diff
@@ -0,0 +1,49 @@
+import React, {Component} from 'react'
+import {bindActionCreators} from 'redux'
+import {connect} from 'react-redux'
+
+import Render from './index.render'
+
+import * as devicesActions from '../../actions/devices'
+import {navigateUp} from '../../actions/router'
+
+export default class DeviceRevoke extends Component {
+  static parseRoute (currentPath) {
+    return {
+      componentAtTop: {
+        title: 'Device revoke',
+        props: {
+          device: currentPath.get('device')
```

Comment:
@MarcoPolo Here's where it gets pushed (maybe you were asking a diff question though?):
```js
  dispatch => {
     return {
       ...bindActionCreators(devicesActions, dispatch),
       showRemoveDevicePage: device => dispatch(routeAppend({path: 'removeDevice', device}))
     }
   }
```